### PR TITLE
Tickets

### PR DESCRIPTION
**Pull request by KorrumzPL**
A ticketing system that makes it easy to support users on the server. It can be divided into two types of categories: creating threads on a channel and creating a channel in a created category, in the `/tickets setup` command.

### CODE REVIEW

**Comment by oceaann** [comment]
> Please note, this system is currently in the early stages of development!
Just mark pr as draft

**Comment by KorrumzPL** [comment]
> > Please note, this system is currently in the early stages of development!
> 
> Just mark pr as draft

I forgot about that, done

File: src/commands/config/addmod.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            id: existingPermission.id,
+            guildId: interaction.guild.id,
+          },
+        },
+        data: {
+          permission: updatedPermission,
+        },
+      });
+    } else {
+      const existingTickets = await client.prisma.permissionSettings.findMany({
```
Comment (oceaann) [inline src/commands/config/addmod.js:40]: What's the point of fetching all the tickets when you can fetch only a last ticket

Comment (KorrumzPL) [reply src/commands/config/addmod.js:40]: None at all. I will correct it in the next commit

Comment (KorrumzPL) [reply src/commands/config/addmod.js:40]: This is the wrong name for the variable. It takes permissions, not tickets.

**Pull request merged**